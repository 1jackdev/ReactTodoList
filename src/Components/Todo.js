import { React } from "react";
import "./Todo.css";

const Todo = ({ id, text, handleRemove }) => {
  const remove = () => handleRemove(id);
  return (
    <>
      <li key={id} className="todo-container">
        {text}
      </li>
      <button type="button" onClick={remove} className="btn">
        X
      </button>
    </>
  );
};

export default Todo;
