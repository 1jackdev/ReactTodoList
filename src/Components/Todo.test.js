import React from "react";
import { render, fireEvent, queryByText } from "@testing-library/react";
import Todo from "./Todo";

it("renders without crashing", () => {
  render(<Todo />);
});

it("matches snapshot", () => {
  const { asFragment } = render(<Todo />);
  expect(asFragment()).toMatchSnapshot();
});
