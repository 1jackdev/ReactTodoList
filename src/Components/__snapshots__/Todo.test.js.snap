// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <li
    class="todo-container"
  />
  <button
    class="btn"
    type="button"
  >
    X
  </button>
</DocumentFragment>
`;
